Delicate-Butterflies/Bangazon-Workforce
# Sb training view edit

Related Issue: #10

Description: Users can view a list of the training programs in start date descending order. A user can also click the edit button. This will present a form to them from which they can edit a training program.

Chai/Mocha Tests File:

PR Testing Steps:
- Fetch branch
- run nodemon
- Check that the programs are in order
- Click the edit button
- Change information and submit
- Verify that the information updated on the programs list

Check list before you make a PR:
- [ x ] comments for JSDOCS
- [ x ] Removed console.logs
